mlflow/tracking/fluent.py

```diff
@@ -52,6 +52,7 @@
 if TYPE_CHECKING:
     import pandas  # pylint: disable=unused-import
     import matplotlib  # pylint: disable=unused-import
+    import matplotlib.figure
```

Comment:
why do we need this import statement?

Reply:
I saw the type definition using it: e.g. `figure: Union["matplotlib.figure.Figure"]...`

Reply:
I see, makes sense!

Reply:
Do we need `# pylint: disable=unused-import` here?